src/DetailsView/Styles/detailsview.scss

```diff
@@ -308,8 +304,18 @@ div.insights-file-issue-details-dialog-container {
     }
     .header-bar {
         padding-left: 12px;
-        position: fixed;
+        position: relative;
         z-index: 1000;
+        display: flex;
+        justify-content: space-between;
+        min-width: 550px;
```

Comment:
I'm not sure the min-width is necessary; even with it, the "Accessibility Insights for Web" text gets split over 2 lines when resizing the window down to its minimum width at 100% zoom, and using the min-width smaller than the 500px min-width of the window means that the settings button slides out of visibility at the minimum window width.

Without min-width, the text still splits unpleasantly, but the gear stays visible, which I think is more important. Maybe the best option would be to omit min-width and also set up the title text to use `text-overflow: ellipsis;`?

Reply:
current production, before bookmarklet
![screenshot in captioned state](https://user-images.githubusercontent.com/376284/70085371-efd97280-15c4-11ea-95b7-0b6afc6c21c4.png)

with current iteration of change, before bookmarklet:
![screenshot in captioned state](https://user-images.githubusercontent.com/376284/70085268-bd2f7a00-15c4-11ea-9459-b50499b797a9.png)

with current iteration of change, after bookmarklet:
![screenshot in captioned state](https://user-images.githubusercontent.com/376284/70085315-d1737700-15c4-11ea-9d2c-2f12e3d0cb9e.png)

this iteration + removing min-width, before bookmarklet:
![screenshot in captioned state](https://user-images.githubusercontent.com/376284/70085545-4d6dbf00-15c5-11ea-8bca-8ac6a1e633c4.png)

this iteration + removing min-width, after bookmarklet:
![screenshot in captioned state](https://user-images.githubusercontent.com/376284/70085446-18616c80-15c5-11ea-91bc-4b5c6275d3a1.png)

Reply:
On further investigation, this issue actually wasn't introduced by this PR; it was introduced by a separate change that's happened since the last production release.